### ISSUE TITLE
Unexpected results in Jacobians and surface emissivity/reflectivity

### ISSUE THREAD

**Issue by cbblanke** [JCSDA/crtm#25]
I am getting some unexpected results and I wonder if I am doing something wrong or there is some other reason.  I am using pyCRTM (with CRTM 2.4.0) but have also verified just using Fortran 90. 

Emissivity and reflectivity don’t sum to 1 (sometimes as much as 1.15) for microwave channels (e.g. GMI).  Is this intentional?

Some temperature Jacobians are negative in places (see figure). From a basic analysis of radiative transfer equations, it would seem that increasing the temperature of a level (and changing nothing else) should always make the TB change positive (or zero).   What am I missing?

Thanks!

![GMI_Jacobians](https://user-images.githubusercontent.com/51972338/167167113-aedd0cc8-ef46-4a1d-ba90-94a93e7198b1.PNG)

**Comment by chengdang** [comment]
@cbblanke May I ask what are the surface types for your profile?

> Some temperature Jacobians are negative in places (see figure).
Are these plots for a microwave channel too? Do you perhaps recall the wavenumber and surface types? 

Could you please provide the input profiles/surface types for us to test and reproduce these results? (for example, profile 37H) Thank you,

**Comment by cbblanke** [comment]
Surface type is ocean.  This is GPM GMI.
See attachment for the complete profile and the channel 7 (37H) Jacobian from the Fortran CRTM run.
[check.txt](https://github.com/JCSDA/crtm/files/8642566/check.txt)

**Comment by chengdang** [comment]
Thanks. Let me do some tests first and I will tag you later for the follow up.

**Comment by chengdang** [comment]
CRTM adopted the FASTEM microwave emissivity model (https://ieeexplore.ieee.org/document/5580050)
@cbblanke To answer part of your question , Mark @quanhualiu gave an explanation on CRTM MW water emissivity model : 

"FASTEM microwave emissivity model has been using at NWP centers for many years. The first version FASTEM-1 uses the equation reflectivity = 1 - emissivity. The performance was not good at NWP center because RTTOV and CRTM treat the surface as "mirror" reflection although the emissivity is calculated with sea roughness. In particular under clear-sky conditions, both RTTOV and CRTM only use one downward radiation at the satellite zenith angle. For relative small zenith angle (< 30 degree), the downwelling radiation is smaller than the integrated downward radiation over all directions because the larger zenith angles are the larger downward radiation.

To overcome the problem, FASTEM 2 use effective surface reflectance to compensate downward radiation. For small zenith angles, the effective reflectance will be large. So do the FASTEM 3, 4, 5, and 6. The FASTER 4 and after outperforms FASTEM-1 for weather forecast. The FASTEM model is limited for zenith angles < 65 degree now. You can observe that the sum of the CRTM emissivity + reflectivity (vertical polarization) decreases as the zenith/scan angles increase. The explanation will be more complicated for horizontal polarization.

In short, the CRTM ocean MW reflectivity is an "effective" value for compensating the downward radiation."
